gulpfile.js

```diff
@@ -7,7 +8,13 @@ gulp.task('clean', function(done) {
   done();
 });
 
-gulp.task('default', ['clean'], function() {
+gulp.task('lint', function() {
+  return gulp.src(['./*.js', './static/*.js'])
```

Comment:
The previous comment is cut off because of markdown I guess, to actually see it click `edit`.